modules/openapi-generator/src/main/resources/go/api.mustache

```diff
@@ -88,7 +98,11 @@ func (r {{#structPrefix}}{{&classname}}{{/structPrefix}}Api{{operationId}}Reques
 Deprecated
 {{/isDeprecated}}
 */
-func (a *{{{classname}}}Service) {{{nickname}}}(ctx _context.Context{{#pathParams}}, {{paramName}} {{{dataType}}}{{/pathParams}}{{#bodyParams}}, {{paramName}} *{{{dataType}}}{{/bodyParams}}) {{#structPrefix}}{{&classname}}{{/structPrefix}}Api{{operationId}}Request {
+func (a *{{{classname}}}Service) {{{nickname}}}(ctx _context.Context{{#pathParams}}, {{paramName}} {{{dataType}}}{{/pathParams}}{{#bodyParams}}, {{paramName}} *{{{dataType}}}{{/bodyParams}}, accept ...string) {{#structPrefix}}{{&classname}}{{/structPrefix}}Api{{operationId}}Request {
```

Comment:
This mechanism could use the standard http [Header](https://pkg.go.dev/net/http#Header) structure to pass in generic headers, and allow them to be passed/over-ridden.